Description: Removes problematic unicode chars in comments
 Under some circonstances, using Python 3.2, the call of:
  python setup.py install
 would produce this:
 Traceback (most recent call last):
   File "setup.py", line 86, in <module>
     version=read_version(),
   File "setup.py", line 46, in read_version
     finder.visit(ast.parse(local_file('httpretty', '__init__.py')))
   File "setup.py", line 78, in <lambda>
     open(os.path.join(os.path.dirname(__file__), *f)).read()
   File "/usr/lib/python3.2/encodings/ascii.py", line 26, in decode
     return codecs.ascii_decode(input, self.errors)[0]
 UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 133: ordinal not in range(128)
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-04-13

--- python-httpretty-0.8.0.orig/readme.rst
+++ python-httpretty-0.8.0/readme.rst
@@ -483,7 +483,7 @@ License
 ::
 
     <HTTPretty - HTTP client mock for Python>
-    Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+    Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 
     Permission is hereby granted, free of charge, to any person
     obtaining a copy of this software and associated documentation
--- python-httpretty-0.8.0.orig/tests/functional/test_bypass.py
+++ python-httpretty-0.8.0/tests/functional/test_bypass.py
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 
 # <httpretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
--- python-httpretty-0.8.0.orig/tests/functional/base.py
+++ python-httpretty-0.8.0/tests/functional/base.py
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 
 # <HTTPretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
--- python-httpretty-0.8.0.orig/tests/functional/testserver.py
+++ python-httpretty-0.8.0/tests/functional/testserver.py
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 #
 # <HTTPretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
--- python-httpretty-0.8.0.orig/tests/functional/__init__.py
+++ python-httpretty-0.8.0/tests/functional/__init__.py
@@ -1,7 +1,7 @@
 # #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # <HTTPretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
--- python-httpretty-0.8.0.orig/tests/functional/test_requests.py
+++ python-httpretty-0.8.0/tests/functional/test_requests.py
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 
 # <HTTPretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
--- python-httpretty-0.8.0.orig/tests/functional/test_httplib2.py
+++ python-httpretty-0.8.0/tests/functional/test_httplib2.py
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 
 # <HTTPretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
--- python-httpretty-0.8.0.orig/tests/functional/test_urllib2.py
+++ python-httpretty-0.8.0/tests/functional/test_urllib2.py
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 
 # <HTTPretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
--- python-httpretty-0.8.0.orig/tests/functional/test_debug.py
+++ python-httpretty-0.8.0/tests/functional/test_debug.py
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 
 # <HTTPretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
--- python-httpretty-0.8.0.orig/tests/unit/__init__.py
+++ python-httpretty-0.8.0/tests/unit/__init__.py
@@ -1,7 +1,7 @@
 # #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # <HTTPretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
--- python-httpretty-0.8.0.orig/tests/unit/test_httpretty.py
+++ python-httpretty-0.8.0/tests/unit/test_httpretty.py
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 
 # <HTTPretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
--- python-httpretty-0.8.0.orig/httpretty/__init__.py
+++ python-httpretty-0.8.0/httpretty/__init__.py
@@ -1,7 +1,7 @@
 # #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # <HTTPretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
--- python-httpretty-0.8.0.orig/httpretty/compat.py
+++ python-httpretty-0.8.0/httpretty/compat.py
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 #
 # <HTTPretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
--- python-httpretty-0.8.0.orig/httpretty/http.py
+++ python-httpretty-0.8.0/httpretty/http.py
@@ -1,7 +1,7 @@
 # #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # <HTTPretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
--- python-httpretty-0.8.0.orig/httpretty/core.py
+++ python-httpretty-0.8.0/httpretty/core.py
@@ -1,7 +1,7 @@
 # #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # <HTTPretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
--- python-httpretty-0.8.0.orig/httpretty/errors.py
+++ python-httpretty-0.8.0/httpretty/errors.py
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 #
 # <HTTPretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
--- python-httpretty-0.8.0.orig/httpretty/utils.py
+++ python-httpretty-0.8.0/httpretty/utils.py
@@ -1,7 +1,7 @@
 # #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # <HTTPretty - HTTP client mock for Python>
-# Copyright (C) <2011-2013>  Gabriel Falcão <gabriel@nacaolivre.org>
+# Copyright (C) <2011-2013>  Gabriel Falcao <gabriel@nacaolivre.org>
 #
 # Permission is hereby granted, free of charge, to any person
 # obtaining a copy of this software and associated documentation
